r/R/anime.R

```diff
@@ -42,13 +69,17 @@ anime <- function(source, target, distance_tolerance = 10, angle_tolerance = 5)
   init_anime(source, target, distance_tolerance, angle_tolerance)
 }
 
-
+#' @param x An `anime` object
```

Comment:
Since this is an s3 method in base we shouldn't documented it. The export is so that R knows we are registering a new method

Reply:
This was for the checks to pass. Suggestion: don't export.

Reply:
They need to be exported for the s3 methods to be registered. You should run devtools::document() so that the methods are registered in the NAMESPACE